spec/shared_contexts/with_stubbed_search.rb

```diff
@@ -33,13 +33,6 @@
   end
 
   before do
-    stubbed_search_results.each do |result|
```

Comment:
This was masking the fact that some pages should have been received. The system specs now are configured as if working in a world without SNS and therefore failsafe by default.